drivers/network/tcpip/tcpip/iinfo.c

```diff
@@ -84,7 +95,11 @@ TDI_STATUS InfoTdiQueryGetInterfaceMIB(TDIEntityID ID,
                     case NdisPhysicalMediumWirelessWan:
 #endif
                     case NdisPhysicalMediumNative802_11:
+#if (_WIN32_WINNT >= _WIN32_WINNT_VISTA)
                         OutData->if_type = IF_TYPE_IEEE80211;
+#else
+                        OutData->if_type = MIB_IF_TYPE_ETHERNET;
+#endif
```

Comment:
I believe the whole `NdisPhysicalMediumNative802_11` case (aka Native 802.11) is specific to NT6+ / NDIS 6 - https://learn.microsoft.com/en-us/previous-versions/windows/hardware/wireless/native-802-11-wireless-lan

Reply:
Ah, no, sorry! My mistake! `NdisPhysicalMediumWirelessLan` is actually returned in that case, as I remember from my tests (from the case above with comment). And that one is not declared to supported only by NDIS 6.0 (so hence it's valid for NDIS 5.2 too):
<img width="852" height="51" alt="Screenshot 2025-08-04 at 14-56-11 OID_GEN_PHYSICAL_MEDIUM - Windows drivers Microsoft Learn" src="https://github.com/user-attachments/assets/8882112f-ab3b-4f45-90eb-a37236f65dd7" />
Then this one can be safely ifdefed I guess.

Reply:
Done.